src/app/components_ngrx/infotip/infotip.component.ts

```diff
@@ -15,9 +20,26 @@ import { Component, Input, HostListener } from '@angular/core';
              </span>`
 })
 
-export class InfotipComponent {
-  @Input() infotipText: string;
+export class InfotipComponent implements OnInit{
+  @Input('uuid') set uuidInput(val: string) {
+    this.infotipsStore.subscribe(i => {
+      console.log(i,'infotips');
+      this.infotipText = i[val] ? i[val].en_EN : val;
+    });
+  };
+  infotipText: string;
+  infotipsStore: Observable<InfotipState>;
 
+  constructor(private store: Store<AppState>) {
+    this.infotipsStore = this.store
+                         .select('listPage')
+                         .select('infotips');
+  }
+
+  ngOnInit() {
+    this.store.dispatch(new InfotipActions.Get());
```

Comment:
Why do you have to dispatch this action every time once info tip component is initiated?

Reply:
Not every time but only once the component is initiated